src/content/docs/dropins/cart/containers/cart-summary-grid.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: CartSummaryGrid container
+description: Learn about the CartSummaryGrid container.
+---
+
+import Diagram from '@components/Diagram.astro';
+import Callouts from '@components/Callouts.astro';
+import OptionsTable from '@components/OptionsTable.astro';
+import Aside from '@components/Aside.astro';
+import { Steps } from '@astrojs/starlight/components';
+import CodeInclude from '@components/CodeInclude.astro';
+import cartModel from '../_includes/cartModel.ts?raw';
+
+The `CartSummaryGrid` container manages and displays the contents of the shopping cart in a grid layout. Its state is managed by the `CartModel` interface, which contains the cart's initial data and is passed down to any child components.
+
+<Diagram caption="CartSummaryGrid container">
+  ![CartSummaryGrid container](@images/dropins/cart/cart-summary-grid-small.png)
+</Diagram>
+
+## Configurations
+
+The `CartSummaryGrid` container provides the following configuration options:
+
+<OptionsTable
+  compact
+  options={[
+    ['Option', 'Type', 'Req?', 'Description'],
+    ['children', 'CartModel', 'Yes', 'Child elements to be rendered inside the container.'],
+    ['initialData', 'string', 'Yes', 'Initial cart data to preload the component. Defaults to null'],
+    ['routeProduct', 'function', 'No', 'Callback function that returns a product.'],
+    ['routeEmptyCartCTA', 'function', 'No', 'Callback function that returns an empty cart.'],
+  ]}
+/>
+
+The `CartModel` object has the following shape:
+
+<CodeInclude code={cartModel} lang="ts" frame="none" />
+
+## Example configuration
+
+The following example demonstrates how to render the `CartSummaryGrid` container with the `routeProduct` and `routeEmptyCartCTA` callbacks:
+
+```js
```

Comment:
**Suggestion**: I see the container import statements is documented on the installation page, but if you want to include it here too, it would look like this:

```js
import { render as provider } from '@dropins/storefront-cart/render.js';

// Dropin Containers
import CartSummaryGrid from '@dropins/storefront-cart/containers/CartSummaryGrid.js';
```

**Suggestion**: Also, we could potentially remove the extra leading spaces in front of the code snippet.